app/src/main/kotlin/io/homeassistant/companion/android/settings/sensor/views/SensorListView.kt

```diff
@@ -42,24 +36,15 @@ fun SensorListView(
                 key = manager.id(),
             ) {
                 if (currentSensors.any()) {
-                    Column(
+                    Box(
                         modifier = Modifier
                             .background(MaterialTheme.colors.background)
                             .fillMaxWidth(),
                     ) {
-                        Row(
-                            modifier = Modifier
-                                .height(48.dp)
-                                .padding(start = 72.dp, end = 16.dp),
-                            verticalAlignment = Alignment.CenterVertically,
-                        ) {
-                            Text(
-                                text = stringResource(manager.name),
-                                style = MaterialTheme.typography.body2,
-                                fontWeight = FontWeight.Bold,
-                                color = MaterialTheme.colors.primary,
-                            )
-                        }
+                        SettingsSubheader(
+                            text = stringResource(manager.name),
+                            paddingForIcon = true,
```

Comment:
```suggestion
                            modifier = Modifier.padding(start = 72.dp),
```
I guess this would respect more the way of creating reusable composable components. It would make the `SettingsSubheader` agnostic from the Icon.

Reply:
> I guess this would respect more the way of creating reusable composable components. It would make the SettingsSubheader agnostic from the Icon.

While reusing is nice, this is supposed to be quite a specific component and the current parameter is much more descriptive than you having to provide 16/72 dp. We're not building a toolkit where everything has to be customizable.

I've adjusted it to provide a modifier slot, but also added a parameter for the text padding with named options (similar to something like `ButtonDefaults` in Compose).